components/AssistantPanel.tsx

```
@@ -0,0 +1,57 @@
+        <Text style={{ fontSize: 16, fontWeight: '700' }}>Coach IA</Text>
+        <Switch value={assistantEnabled} onValueChange={() => dispatch(toggleAssistant())} />
+      </View>
+      <Pressable
+        onPress={onAsk}
+        style={{ padding: 10, backgroundColor: '#111827', borderRadius: 10 }}
+      >
+        <Text style={{ color: 'white', textAlign: 'center' }}>Demander une suggestion</Text>
+      </Pressable>
+      <ScrollView style={{ marginTop: 10 }}>
+        {suggestions.length === 0 ? (
+          <Text style={{ color: '#6b7280' }}>Aucune suggestion pour l'instant.</Text>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)